modules/dnn/src/tengine4dnn/src/tengine_graph_convolution.cpp

```diff
@@ -136,22 +135,23 @@ int create_conv_node(graph_t graph, const char* node_name, const char* input_nam
     set_node_attr_int(conv_node, "pad_h1", &pad_h1);
     set_node_attr_int(conv_node, "pad_w1", &pad_w1);
     set_node_attr_int(conv_node, "output_channel", &outch);
+    set_node_attr_int(conv_node, "input_channel", &inch);
     set_node_attr_int(conv_node, "group", &group);
     set_node_attr_int(conv_node, "dilation_h", &dilation_h);
     set_node_attr_int(conv_node, "dilation_w", &dilation_w);
-    set_node_attr_int(conv_node, "activation", &activation);
+  //  set_node_attr_int(conv_node, "activation", &activation);
 
     release_graph_node(conv_node);
 
     return 0;
 }
 
-graph_t create_conv_graph(float *input_data, int inch, int group, int in_h, int in_w,
+graph_t create_conv_graph(const char * layer_name, float *input_data, int inch, int group, int in_h, int in_w,
                         float *output_data, int outch, int out_h, int out_w,
                         int kernel_h, int kernel_w,
                         int stride_h,int stride_w,
                         int pad_h, int pad_w,  int dilation_h, int dilation_w, int activation,
-                        float * teg_weight , float * teg_bias , std::string padMode)
```

Comment:
the common typographic convention in text and programs is that there should not be a space before punctuation, and there should be space after: `float * teg_weight , float ...` => `float* teg_weight, float ...`

Reply:
done. 
You're a perfect coder. Learn from you.